src/terminal/input/terminalInput.hpp

```diff
@@ -92,7 +93,9 @@ namespace Microsoft::Console::VirtualTerminal
         // storage location for the leading surrogate of a utf-16 surrogate pair
         std::optional<wchar_t> _leadingSurrogate;
 
-        til::enumset<Mode> _inputMode{ Mode::Ansi };
+        WORD _lastVirtualKeyCode{ 0 };
+
+        til::enumset<Mode> _inputMode{ Mode::Ansi, Mode::AutoRepeat };

```

Comment:
Is this another instance where Terminal can receive a reset/state toggle for `DECARM` and end up in a torn state ala #13968?

Reply:
I don't think so. In the `DECAC1` case we needed Terminal to have the mode permanently enabled, but that's not the case here. We're always passing through the input modes, so Terminal should always be in sync with conhost.

The other thing is that Terminal would assumedly be using Win32InputMode most of the time, so the auto repeat option would typically not have any effect on that side.